docs/examples/configuring-parallel-tests-with-code-climate.md

```diff
@@ -47,8 +48,8 @@ blocks:
           commands:
             - checkout
             # fetching partial tests from S3
-            - aws s3 sync coverage/ "s3://my_coverage_bucket/$SEMAPHORE_PROJECT_NAME/$SEMAPHORE_GIT_BRANCH/coverage/$SEMAPHORE_WORKFLOW_ID/"
+            - aws s3 sync "s3://my_coverage_bucket/$SEMAPHORE_PROJECT_NAME/$SEMAPHORE_GIT_BRANCH/coverage/$SEMAPHORE_WORKFLOW_ID/" coverage/
```

Comment:
So upon running this command, we were confused why our CI was failing. I think it's because we expect aws to download all coverage files from s3 into a local `coverage/` folder. 

The correct command for that was to interchange the parameters. `s3 link` being the origin and `coverage` being the destination.